frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -186,10 +186,10 @@ local footerTextGeneratorMap = {
                 local aux_batt_lvl = powerd:getAuxCapacity()
                 is_charging = powerd:isAuxCharging()
                 -- Sum both batteries for the actual text
-                batt_lvl = main_batt_lvl + aux_batt_lvl
                 -- But average 'em to compute the icon...
+                batt_lvl = math.floor((main_batt_lvl + aux_batt_lvl) / 2 + 0.5)
```

Comment:
Isn't this exactly the point in this PR. Tweaking the battery level, so that the shown capacity (average) matches the threshold (average). 
Straight forward: calculate the level (average on two batteries) and show or hide it.